views/appeal-application/home-office/appeal-late.njk

```diff
@@ -13,7 +13,7 @@
   <h1 class="govuk-heading-l">{{ i18n.pages.homeOfficeAppealLate.title }}</h1>
   <p>{{ i18n.pages.homeOfficeAppealLate.description | safe }}</p>
 
-  <form action="/home-office/appeal-late" method="post" enctype="multipart/form-data">
+  <form action="/home-office/appeal-late?_csrf={{ csrfToken }}" method="post" enctype="multipart/form-data">
```

Comment:
Why cant we use here the hidden field? it is related to the 'multipart/form-data'?

Reply:
yes